### PR TITLE
Make configure.sh work with i18n git by requiring POSIX locale.

### PR DESCRIPTION
**Pull request by yeputons**
On some machines a locale is set for git, which makes `git status` and some other commands print localized messages, which in turn breaks `git verify`.

This PR fixes that by forcing all commands run inside `git start`/`git verify` to be run with a "standard" POSIX locale (English).

Here is an example of broken exercises (locale is `ru_RU.UTF-8`, Russian):
```
$ git clone https://gitexercises.fracz.com/git/exercises.git
Клонирование в «exercises»…
remote: Counting objects: 135, done.
remote: Compressing objects: 100% (104/104), done.
remote: Total 135 (delta 28), reused 135 (delta 28)
Получение объектов: 100% (135/135), 31.75 KiB | 2.44 MiB/s, готово.
Определение изменений: 100% (28/28), готово.
$ cd exercises/
$ ./configure.sh
OK
$ git start
Preparing the exercise environment, hold on...
Exercise master started!
Read the README.md for instructions or view them in browser:
http://gitexercises.fracz.com/e/master
$ git verify
You haven't made any progress on exercise master.
Did you forget to commit your changes?
```

The core reason (`git verify` looks for `ahead` substring):
```
$ git status
На ветке master
Ваша ветка опережает «origin/master» на 1 коммит.
  (используйте «git push», чтобы опубликовать ваши локальные коммиты)

нечего коммитить, нет изменений в рабочем каталоге
$ LC_ALL=POSIX git status
On branch master
Your branch is ahead of 'origin/master' by 1 commit.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
```

Can be worked around with:
```
$ LC_ALL=POSIX git verify
Verifying the master exercise. Hold on...
Exercise: master
Status: PASSED
You can see the easiest known solution and further info at:
https://gitexercises.fracz.com/e/master/vq9

Next task: commit-one-file
In order to start, execute: git start next

See your progress and instructions at:
https://gitexercises.fracz.com/c/vq9
```

### CODE REVIEW

File: configure.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
...ef HEAD); exercise=\${1-\$currentBranch}; if [ \$exercise != HEAD ]; then if [ \$exercise != next ]; then if git show origin/\$exercise:start.sh >/dev/null 2>&1 ; then if git checkout -f \$exercise >/dev/null 2>&1 && git reset --hard origin/\$exercise >/dev/null 2>&1 && git clean -fdx >/dev/null ; then if echo \"Preparing the exercise environment, hold on...\" && ./start.sh >/dev/null 2>&1; then echo \"Exercise \$exercise started!\" && echo 'Read the README.md for instructions or view them in browser:' && echo \"http://gitexercises.fracz.com/e/\$exercise\" ; else echo 'Could not execute the start script.' && echo 'Try running the ./start.sh script yourself.' ; fi else echo 'Could not clean the working directory.' && echo 'Make sure that none of the files inside the working directory' && echo 'is used by another process and run git start again.'; fi else echo \"Invalid exercise: \$exercise\" && false; fi else git push origin HEAD:next-exercise 2>&1 | sed -n '/\\*\\*\\*/,/\\*\\*\\*/p' | grep -v '\\*\\*' | sed 's/remote: //g' | xargs git start | grep -v Invalid || echo 'You have passed all exercises!'; fi else echo 'You need to use git start <exercise-name> in detached HEAD'; fi }; f"
```
Comment (yeputons) [inline configure.sh]: This line should only have `local LC_ALL=POSIX;` added in the beginning of the function.

File: configure.sh [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 # Add "git verify" local alias for submitting exercises solutions.
-git config alias.verify "! f() { currentBranch=\$(git rev-parse --abbrev-ref HEAD); exercise=\${1-\$currentBranch}; if [ \$exercise != HEAD ]; then if git show origin/\$exercise:start.sh >/dev/null 2>&1 ; then if git status | grep -E 'ahead|detached' >/dev/null 2>&1 ; then if echo \"Verifying the \$exercise exercise. Hold on...\" && git push -f origin HEAD:\$exercise 2>&1 | sed -n '/\\*\\*\\*/,/\\*\\*\\*/p' | sed 's/remote: //g' | grep -v \"\\*\\*\" ; then : ; else echo 'Solution could not be verified - push failed.' && echo 'Do you have an internet connection?'; fi else echo \"You haven't made any progress on exercise \$exercise.\" && echo 'Did you forget to commit your changes?'; fi else echo \"Invalid exercise: \$exercise\"; fi else echo 'You need to use git verify <exercise-name> in detached HEAD'; fi }; f"
```
Comment (yeputons) [inline configure.sh]: This line should only have `local LC_ALL=POSIX;` added in the beginning of the function.

**Comment by fracz** [comment]
Thank you!

**Comment by fracz** [comment]
Resolved a conflict and merged in #39

**Pull request closed without merging**